src/containers/oracle.js

```diff
@@ -158,70 +155,235 @@ class OraclePage extends React.Component {
+        if (oracle.resultSetterAddress !== selectedWalletAddress) {
+          config.cardInfo.messages.push({
+            text: 'You are not the result setter for this topic and cannot set result.',
+            type: 'warn',
+          });
+        }
+      } else if (status === 'VOTING' && token === BOT) {
+        const relatedOracles = _.filter(getOraclesSuccess, (item) => item.topicAddress === oracle.topicAddress);
+        const centralizedOracle = _.find(relatedOracles, (item) => item.token === QTUM);
+
+        config = {
+          name: 'VOTING',
+          breadcrumbLabel: 'Votting',
+          cardInfo: {
+            steps: {
+              current: 3,
+              value: [{
+                title: 'Topic created',
+                description: `Block No. ${(centralizedOracle && centralizedOracle.blockNum) || ''}`,
+              },
+              {
+                title: 'Betting',
+                description: `Block No. ${(centralizedOracle && centralizedOracle.blockNum + 1) || ''} - ${(centralizedOracle && centralizedOracle.endBlock) || ''}`,
+              },
+              {
+                title: 'Result Setting',
+                description: `Block No. ${(centralizedOracle && centralizedOracle.endBlock + 1) || ''} - ${(centralizedOracle && centralizedOracle.resultSetEndBlock)}`,
+              },
+              {
+                title: 'Voting',
+                description: `Block No. ${oracle.blockNum} - ${oracle.endBlock}`,
+              },
+              ],
+            },
+            messages: [
+              {
+                text: `Consensus Threshold ${oracle.consensusThreshold || 0}. This value indicates the amount of BOT needed to fulfill current voting challenge.`,
+                type: 'default',
+              }, {
+                text: 'BOT tokens are needed for Voting. Upon clicking Confirm, you will need to wait for BOT token to get approved. Those amount will automatically be used to Vote afterwards.',
+                type: 'default',
+              },
+            ],
+          },
+          cardAction: {
+            skipToggle: false,
+            beforeToggle: {
+              btnText: 'Vote',
+            },
+            afterToggle: {
+              showAmountInput: true,
+              btnText: 'Confirm',
+            },
+          },
+        };
+      } else if (status === 'WAITRESULT' && token === BOT) {
+        const relatedOracles = _.filter(getOraclesSuccess, (item) => {
+          console.log(item.topicAddress, oracle.topicAddress);
+          return item.topicAddress === oracle.topicAddress;
+        });
+        const centralizedOracle = _.find(relatedOracles, (item) => item.token === QTUM);
+        const decentralizedOracles = _.orderBy(_.filter(relatedOracles, (item) => item.token === BOT), ['blockNum'], ['asc']);
+
+        config = {
+          name: 'FINALIZING',
+          breadcrumbLabel: 'Voting',
```

Comment:
Finalizing

Reply:
No, finalizing is in the same phase(tab) as Voting. We don't show Finalizing on tabs/breadcrumb navigation.